src/main.rs

```diff
@@ -535,79 +535,80 @@ impl ManParser for Type1 {
         manpage.contains(r#".SH "OPTIONS""#)
     }
 
-    fn parse_man_page(&mut self, manpage: &str) -> Option<String> {
+    fn parse_man_page(&mut self, manpage: &str, cmd_name: &str) -> Option<String> {
+        let options_section_re = regex!(r#"\.SH "OPTIONS"((?s:.)*?)(\.SH|\z)"#);
+        let options_section_matched = options_section_re.find(manpage);
+
+        let mut options_section = options_section_matched.unwrap().as_str();
```

Comment:
Why not
```suggestion
        let mut options_section = &options_section_matched.unwrap_or_default()
```

Reply:
```rust
error[E0599]: no method named `unwrap_or_default` found for type `std::option::Option<regex::re_unicode::Match<'_>>` in the current scope
   --> src/main.rs:542:60
    |
542 |         let mut options_section = &options_section_matched.unwrap_or_default();
    |                                                            ^^^^^^^^^^^^^^^^^ method not found in `std::option::Option<regex::re_unicode::Match<'_>>`
    |
    = note: the method `unwrap_or_default` exists but the following trait bounds were not satisfied:
            `regex::re_unicode::Match<'_> : std::default::Default`
```

Reply:
Ah, makes sense. I thought it was a string lol